runtime/foucoco/src/lib.rs

```diff
@@ -634,6 +631,7 @@ impl pallet_collective::Config<CouncilCollective> for Runtime {
 	type MaxMembers = CouncilMaxMembers;
 	type DefaultVote = pallet_collective::PrimeDefaultVote;
 	type WeightInfo = pallet_collective::weights::SubstrateWeight<Runtime>;
+	type SetMembersOrigin = EnsureRoot<AccountId>;
```

Comment:
following [Astar's](https://github.com/AstarNetwork/Astar/blob/temp/polkadot-v0.9.40/runtime/shibuya/src/lib.rs#L892) and [Acala's](https://github.com/AcalaNetwork/Acala/blob/master/runtime/mandala/src/lib.rs#L373)